R/derive_var_anrind.R

```diff
@@ -1,9 +1,11 @@
 #' Derive Reference Range Indicator
 #'
 #' @param dataset The input dataset
+#' @param use_a1hia1lo Boolean value indicating whether to use `A1H1` and `A1LO` in
+#' the derivation of `ANRIND`. Defaults to TRUE.
```

Comment:
Guidance moving forward is to remove mention of default values since it is in the function header.